lib/taskana-core/src/main/java/pro/taskana/impl/TimeIntervalReportBuilderImpl.java

```diff
@@ -64,25 +68,25 @@ public B inWorkingDays() {
 
     @Override
     public B workbasketIdIn(List<String> workbasketIds) {
-        this.workbasketIds = workbasketIds;
+        this.workbasketIds = new ArrayList<>(workbasketIds);
```

Comment:
have a look into Collections.unmodifiableList()

Reply:
Collections.unmodifiableList() makes the current reference unmodifiable, but the given list can still be modified.